src/app.rs

```diff
@@ -658,6 +706,26 @@ impl eframe::App for BdiffApp {
 }
 
 impl BdiffApp {
+    fn overwrite_modal(&mut self, modal: &Modal) {
+        modal.show(|ui| {
+            modal.title(ui, "Overwrite previous config");
+            ui.label("By saving, you are going to overwrite previous configuration.");
```

Comment:
I think I'd prefer "By saving, you are going to overwrite the existing configuration file at "`PATH`/bdiff.json"